e2e/Dockerfile.tensorboard

```diff
@@ -0,0 +1,6 @@
+FROM gcr.io/kubeflow/jupyterhub-k8s:1.0.1
```

Comment:
I couldn't get it to work with the stock TF image. It might be a Tensorboard 1.5 issue or the way the image is built. But I just kept getting "No dashboards are active for the current data set." The combination of of TF 1.5 and Tensorboard 1.6 works fine.